### ISSUE TITLE
Test failure: invalid level[OFF] for logger[]

### ISSUE THREAD

**Issue by jordansissel** [elastic/logstash#8518]
Failing for the past 12 days

Examples: 

* https://logstash-ci.elastic.co/job/elastic+logstash+5.6+multijob-integration-1/50/console
* https://logstash-ci.elastic.co/job/elastic+logstash+5.6+multijob-integration-pq-2/50/console
* https://logstash-ci.elastic.co/job/elastic+logstash+5.6+multijob-integration-2/51/console

```
19:46:48 ArgumentError:
19:46:48   invalid level[OFF] for logger[]
19:46:48 # ./specs/01_logstash_bin_smoke_spec.rb:1:in `(root)'
19:46:48 # ./specs/01_logstash_bin_smoke_spec.rb:6:in `(root)'
```

The stack trace is confusing and doesn't seem to really have meaningful details.

**Comment by original-brownbear** [comment]
I've also seen this in plugin tests when ran against `5.6.4` lately. Could be some dependency issue with the Logger? Looking at https://github.com/elastic/logstash/pull/8470/files#diff-302f1511fd356ce35640f85cc8f5445aR5 here in particular.

**Comment by jakelandis** [comment]
Just ran into this for the JDBC input plugin https://travis-ci.org/logstash-plugins/logstash-input-jdbc/jobs/292245059 , but only for the 1.7 matrix build.

**Comment by jakelandis** [comment]
Easily reproduced locally
```
docker build  -t logstash-integration-tests .
docker run -it --rm logstash-integration-tests ci/integration_tests.sh specs/01_logstash_bin_smoke_spec.rb
```
👀 now...

**Comment by jakelandis** [comment]
[Tthis commit](https://github.com/elastic/logstash-devutils/commit/088721764a93896beda214a0ef5b9a6fec62cf0b) removes code from devutil that initializes the logging framework which is valid for 6.0+ since there were many changes to early lifecycle of the logging subsystem making the devutil code redundant (and possibly buggy?). 

However, it appears that 5.6 still needs this initialization. 

The fix here will be to pin the integration tests devutils to 1.3.4, the last release prior to the commit referenced above.

**Comment by jakelandis** [comment]
Re-opened issue the initial fix only fixed the 5.6 build branch via CI. Other build types are also broken in this manner.
* Plugins, the default build (outside of the matrix) relies on the latest devutils which depends on 5.5.1 of logstash core. 
* Developer builds, without a bit of gymnastics this runs into the same issue as the default plugin build. 

The root cause of this issue is that the latest devutils is no longer compatible with 5.x but until major changes are made w/r/t devutils and versions, this will cause issues unless we fix the root cause. 

The new fix here will be a new devutils release that is compatible with 5.x/6.x and Windows.
